src/components/menu/js/menuController.js

```diff
@@ -222,4 +225,42 @@ function MenuController($mdMenu, $attrs, $element, $scope, $mdUtil, $timeout, $r
       throw Error('Invalid offsets specified. Please follow format <x, y> or <n>');
     }
   };
+  
+  /**
+   * Get value from element attributes
+   *
+   * @param {String} attrKey Attribute key
+   * @param {object} defaultValue If vaalue is presented, it will be converted to default value type
+   * @returns {object} defaultValue or attribute value converted to defaultValue type
+   */
+  function getAttr(attrKey, defaultValue) {
+    var result = defaultValue;
+    if(angular.isDefined($attrs[attrKey]))
+      result = parseTypeOf(typeof defaultValue, $attrs[attrKey]);
+
+    return result;
+  }
+
+  /**
+   * Convert value to type
+   *
+   * @param {String} type Object type
+   * @param {object} value Value to be converted
+   * @returns {*} Converted value to type
+   */
+  function parseTypeOf(type, value) {
```

Comment:
I don't like those *two* new methods. They're in my opinion not really necessary.